Assignment1/Assignment1_QNo1/SolAssignmentNo1_QNo1.rb

```diff
@@ -0,0 +1,25 @@
+# => 1. Find out how to convert the string "7.13" into the number 7.13
+# => author : Hrisikesh Singha Chowdhury
+
+# => definition of custom class
+class CustomClass
+  # => Fn to implement buisness logic
+  def cvt_str_to_int
+    puts "Enter another string"
+    str = gets.chomp
+    str_output = str.chomp.to_f
```

Comment:
@HrisikeshSinghaChowdhury Why do we need chomp here?

Reply:
@HrisikeshSinghaChowdhury You have already used chomp once in the previous statement then why need again?
Also chomp is not for removing trailing white space, it will remove carriage return characters (i.e. \n, \r, and \r\n)
To remove the white space there is another method called `strip`

Reply:
ok